src/content/api/cli.md

```diff
@@ -272,6 +272,20 @@ webpack --analyze
 
 W> Make sure you have `webpack-bundle-analyzer` installed in your project else CLI will prompt you to install it.
 
+## Progress
+
+To check the progress of any webpack compilation you can use the `--progress` flag.
+
+```sh
```

Comment:
```suggestion
```bash
```
Let's use `bash` here as `sh` is not supported in PrismJS https://travis-ci.org/github/webpack/webpack.js.org/jobs/741912024#L570

Reply:
then we probably need to update it at more places too.